docs/google/calibration.md

```diff
@@ -0,0 +1,217 @@
+signals.  These signals must be interpreted as signifying one state or the
+other.  Since these analog signals are continuous distributions, there will be
+some statistical overlap in the two distributions that would be theoretically
+impossible to distinguish.  This is classified as the separation error, and is
+calculated by fitting Gaussian distributions to the signals prepared in the
+|0⟩ state and |1⟩ state and calculating the overlap between the two
+distributions.
+
+### Isolated 1 qubit randomized benchmark error: 
+*   Metric key: single_qubit_rb_average_error_per_gate
+*   Metric key: single_qubit_rb_pauli_error_per_gate
+*   Metric key: single_qubit_rb_incoherent_error_per_gate
+
+Single qubit gate error is estimated using randomized benchmarking by taking
+sequences of varying length of the 24 gates within the Clifford group
+(those that preserve the Pauli group under conjugation) then applying then
+applying the inverse of the unitary equivalent to the executed gate sequence.
+The result of the total sequence should always be the identity (|0⟩ state).
+The final error is measured and compared against this
+state to produce the total error.  This error is calculated for one qubit at a
+time while all other qubits on the device are idle (isolated).  See the
+above section for descriptions of total versus purity error.
+
+More information about randomized benchmarking can be found in section 6.3
+(page 120) of this
+[thesis](https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf).
+
+### T1 
+*   Metric key: single_qubit_idle_t1_micros
+
+The T1 of a qubit represents the time constant of the exponential decay of a
+qubit in the excited |1⟩ state to naturally decay into the ground |0⟩ state.
+This is calculated by preparing an excited state with a microwave pulse
+(a.k.a. an X gate), measured after a variety of decay times.
+
+An exponential curve is then fit to the resulting data to determine the T1 time,
+which is reported in microseconds.
+
+### 2-qubit Isolated XEB error
+*   Metric key: two_qubit_sqrt_iswap_gate_xeb_cycle_average_error_per_cycle
+*   Metric key: two_qubit_sqrt_iswap_gate_xeb_cycle_pauli_error_per_cycle
+*   Metric key: two_qubit_sycamore_gate_xeb_cycle_incoherent_error_per_cycle
+*   Metric key: two_qubit_sycamore_gate_xeb_cycle_average_error_per_cycle
+*   Metric key: two_qubit_sycamore_gate_xeb_cycle_pauli_error_per_cycle
+*   Metric key: two_qubit_sycamore_gate_xeb_cycle_incoherent_error_per_cycle
+
+Two qubit error is primarily characterized by applying cross-entropy
+benchmarking (XEB).  This procedure consists of performing a "cycle" of a
+random one-qubit gate on each qubit followed by the two qubit entangling gate.
+The resulting distribution is analyzed and compared to the expected distribution
+using cross entropy.  The value reported is the error rate per cycle (both
+the 1 qubit gates as well as the 2 qubit gate) is incorrect.
```

Comment:
The "is incorrect" seems out of place. Remove?

Reply:
Done.